library/modules/Items.cpp

```diff
@@ -1241,250 +1064,183 @@ df::proj_itemst *Items::makeProjectile(df::item *item) {
-        else
+            if (auto def = df::itemdef_instrumentst::find(item_subtype))
+                value = def->value;
+            break;
+        case WEAPON:
             value = 10;
-        break;
-
-    case item_type::INSTRUMENT:
-        if (size_t(item_subtype) < world->raws.itemdefs.instruments.size())
-            value = world->raws.itemdefs.instruments[item_subtype]->value;
-        else
+            if (auto def = df::itemdef_weaponst::find(item_subtype))
+                value = def->value;
+            break;
+        case ARMOR:
             value = 10;
-        break;
-
-//  case item_type::ROCK:
-    default:
-        return 0;
+            if (auto def = df::itemdef_armorst::find(item_subtype))
+                value = def->value;
+            break;
+        case SHOES:
+            value = 5;
+            if (auto def = df::itemdef_shoesst::find(item_subtype))
+                value = def->value;
+            break;
+        case SHIELD:
+            value = 10;
+            if (auto def = df::itemdef_shieldst::find(item_subtype))
+                value = def->value;
+            break;
+        case HELM:
+            value = 10;
+            if (auto def = df::itemdef_helmst::find(item_subtype))
+                value = def->value;
+            break;
+        case GLOVES:
+            value = 5;
+            if (auto def = df::itemdef_glovesst::find(item_subtype))
+                value = def->value;
+            break;
+        case AMMO:
+            value = 1;
+            if (auto def = df::itemdef_ammost::find(item_subtype))
+                value = def->value;
+            break;
+        case PANTS:
+            value = 10;
+            if (auto def = df::itemdef_pantsst::find(item_subtype))
+                value = def->value;
+            break;
+        case TRAPCOMP:
+            value = 10;
+            if (auto def = df::itemdef_trapcompst::find(item_subtype))
+                value = def->value;
+            break;
+        case TOOL:
+            value = 10;
+            if (auto def = df::itemdef_toolst::find(item_subtype))
+                value = def->value;
+            break;
+        case FISH:
+        case FISH_RAW:
+        case EGG:
+            value = 2;
+            if (auto craw = Units::getCasteRaw(mat_type, mat_subtype)) {
+                mat_type = craw->misc.bone_mat;
+                mat_subtype = craw->misc.bone_matidx;
+            }
+            break;
+        case VERMIN:
+            if (auto craw = Units::getCasteRaw(mat_type, mat_subtype))
+                value = craw->misc.petvalue / 2;
+            return value ? value : 1;
+        case PET:
+            if (auto craw = Units::getCasteRaw(mat_type, mat_subtype))
+                value = craw->misc.petvalue;
+            return value;
+        case FOOD:
+            return 10;
+        case CORPSE:
+        case CORPSEPIECE:
+        case REMAINS:
+        case ROCK:
+        default:
```

Comment:
can we remove the `default` case so the compiler warns us if there is an unhandled enum value?